lib/loggly/client.js

```diff
@@ -172,9 +172,12 @@ Loggly.prototype.log = function (msg, tags, callback) {
     : this.tags;
 
   //
-  // Optionally send `X-LOGGLY-TAG` if we have them
+  // Optionally send `X-LOGGLY-TAG` if we have them.
+  // Set the 'X-LOGGLY-TAG' only when we have actually some tag value.
+  // The library receives "400 Bad Request" in response when the
+  // value of 'X-LOGGLY-TAG' is empty string in request header.
   //
-  if (tags) {
+  if (tags && tags.length) {
```

Comment:
Set the **headers['X-LOGGLY-TAG']** property only when we have some actual tag defined into the tag array not undefined.